src/utils/session.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { INestApplication } from '@nestjs/common';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { TypeormStore } from 'connect-typeorm';
+import session from 'express-session';
+import { Repository } from 'typeorm';
+
+import { AuthConfig } from '../config/auth.config';
+import { Session } from '../users/session.entity';
+
+export function createSession(app: INestApplication, authConfig: AuthConfig) {
```

Comment:
```suggestion
export function setupSessionMiddleware(app: INestApplication, authConfig: AuthConfig) {
```
as this function does not create a single session ;)